rllab/envs/dm_control_env.py

```diff
@@ -0,0 +1,113 @@
+import pygame
+import numpy as np
+
+from dm_control import suite
+from dm_control.rl.environment import StepType
+from dm_control.suite.wrappers import pixels
+
+from rllab.envs.base import Env, Step
+from rllab.envs.dm_control_view import DmControlViewer
+from rllab.core.serializable import Serializable
+from rllab.spaces.box import Box
+from rllab.spaces.discrete import Discrete
+'''
+This environment will use dm_control toolkit(https://arxiv.org/pdf/1801.00690.pdf) 
+to train and simulate your models.
+
+Domain name and task name used for dm_control are showed below: (domain name/task name)
+
+ball_in_cup/catch
+cartpole/swingup_sparse
+cartpole/balance_sparse
+cartpole/swingup
+cartpole/balance
+cheetah/run
+finger/turn_hard
+finger/spin
+finger/turn_easy
+fish/upright
+fish/swim
+hopper/stand
+hopper/hop
+humanoid/stand
+humanoid/run
+humanoid/walk
+manipulator/bring_ball
+pendulum/swingup
+point_mass/easy
+reacher/hard
+reacher/easy
+swimmer/swimmer6
+swimmer/swimmer15
+walker/stand
+walker/run
+walker/walk
+'''
+
+
+def _flat_shape(observation):
```

Comment:
You maybe able to use rllab.spaces.Product for this, but I have not tried it with TensorFlow. We can save that for another day.

Reply:
Product seems not work at tf environment.

Reply:
Yeah, I will file an issue.

Reply:
https://github.com/ryanjulian/rllab/issues/32